### PR TITLE
<CounterBadge /> - Revert overrides

### PR DESCRIPTION
**Pull request by dht**
### 🔦 Summary

...

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [x] Cheatsheet
  - [x] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6743/

**Comment by mykas** [comment]
@adicicurel

File: packages/wix-style-react/src/Themes/businessDashboard/components/Sidebar.st.css [hunk 1]
```diff
@@ -32,3 +32,7 @@
 .root::slider:skin(dark)::-webkit-scrollbar-thumb:hover {
   background: value(darkScrollbarThumbHoverColor);
 }
+
+.root CounterBadge:skin(general):dark {
```
Comment (mykas) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Sidebar.st.css]: Maybe you wanted here to do this?
```
.root:skin(dark) CounterBadge:skin(general) {
```

File: packages/wix-style-react/src/Themes/businessDashboard/components/Sidebar.st.css [hunk 2]
```diff
@@ -32,3 +32,7 @@
 .root::slider:skin(dark)::-webkit-scrollbar-thumb:hover {
   background: value(darkScrollbarThumbHoverColor);
 }
+
+.root CounterBadge:skin(general):theme(dark) {
```
Comment (mykas) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Sidebar.st.css:36]: .root:skin(dark) CounterBadge:skin(general) {

**Comment by dht** [comment]
@mykas 
Following a zoom with @adicicurel and @shlomitc we decided to do those changes (which are A/B tested) in the business-manager.
This PR is meant to revert any changes already made regarding the <CounterBadge /> in the newDashboard theme.
Shlomi explained why the scope for those overrides was too narrow:
- Dark theme
- General skin
- CounterBadges which are only within the Sidebar
And why skins are the right tool for addressing different component surroundings (sidebar, light/dark background) and not custom cascading overrides.

**Review by adicicurel — APPROVED** [review]

**Review by mykas — APPROVED** [review]

**Pull request merged**